codes/classical/Cycle Codes.yml

```diff
@@ -0,0 +1,34 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+
+
+# code id, physical, logical are all lower case
+code_id: cycle
+physical: bits
+logical: bits
+
+
+name: 'cycle code of graphs '
+introduced: '\cite{https://ieeexplore.ieee.org/document/394711}'
+
+
+description: 'Let \(G\) be a finite, undirected, connected graph without loops and multiple edges.The graph \(G\) has (\r\) vertices and \(n\) edges. Its parity check matrix (\H\) is defined as follows.(\H\) is an (\ r \times n\) matrix whose rows are indexed by the vertices of \(G\) and whose columns are indexed by the edges of \(G\), and such that \(H_{ij}=1\) if vertex \(i\) belongs to the edge \(j\) and \(H_{ij}=0\) otherwise.'
+
+
```

Comment:
I quickly skimmed the paper you cite, it seems to deal with threshold behavior for the error probability. This should be mentioned in protection field. If asymptotic, it should be mentioned in rate field.

Reply:
The cycle code corrects up to [(d-1)/2] Hamming errors, where d is the length of the shortest cycle in the graph.
The threshold probability paper may be mentioned, but the above is better as the main entry under "protection".